lib/bourbon.rb

```diff
@@ -1,17 +1,9 @@
-# CodeKit needs relative paths
```

Comment:
@bdkjones Hello! We’re looking to merge these changes in, but we are making an assumption here that this stuff is no longer needed. Our sister library [Neat](https://github.com/thoughtbot/neat/)—which is also part of CodeKit—doesn’t appear to have any of this. If you have a chance, I’d love a quick review, so we don’t bust anything!